srcpkgs/python3-ansible-lint/template

```diff
@@ -1,34 +1,38 @@
- --deselect test/test_cli_role_paths.py::test_run_role_name_with_prefix
- --deselect test/test_cli_role_paths.py::test_run_role_name_from_meta
+make_check_args="--deselect src/ansiblelint/rules/no_loop_var_prefix.py::test_no_loop_var_prefix
+ --deselect src/ansiblelint/rules/schema.py::test_schema
+ --deselect test/test_cli_role_paths.py::test_run_inside_role_dir
  --deselect test/test_cli_role_paths.py::test_run_invalid_role_name_from_meta
- --deselect test/test_cli_role_paths.py::test_run_role_three_dir_deep
+ --deselect test/test_cli_role_paths.py::test_run_multiple_role_path_no_trailing_slash
+ --deselect test/test_cli_role_paths.py::test_run_multiple_role_path_with_trailing_slash
  --deselect test/test_cli_role_paths.py::test_run_playbook
+ --deselect test/test_cli_role_paths.py::test_run_role_name_from_meta
  --deselect test/test_cli_role_paths.py::test_run_role_name_invalid
+ --deselect test/test_cli_role_paths.py::test_run_role_name_with_prefix
+ --deselect test/test_cli_role_paths.py::test_run_role_three_dir_deep
  --deselect test/test_cli_role_paths.py::test_run_single_role_path_no_trailing_slash_module
- --deselect test/test_cli_role_paths.py::test_run_inside_role_dir
- --deselect test/test_cli_role_paths.py::test_run_single_role_path_with_trailing_slash
- --deselect test/test_cli_role_paths.py::test_run_multiple_role_path_no_trailing_slash
+ --deselect test/test_cli_role_paths.py::test_run_single_role_path_no_trailing_slash_script
  --deselect test/test_cli_role_paths.py::test_run_single_role_path_with_roles_path_env
- --deselect test/test_verbosity.py::test_default_verbosity
- --deselect test/test_task_includes.py::test_included_tasks
- --deselect test/test_utils.py::test_cli_auto_detect
- --deselect test/test_list_rules.py::test_list_rules_includes_opt_in_rules
- --deselect test/test_formatter_json.py::test_code_climate_parsable_ignored
+ --deselect test/test_cli_role_paths.py::test_run_single_role_path_with_trailing_slash
+ --deselect test/test_eco.py::test_eco
  --deselect test/test_examples.py::test_custom_kinds
- --deselect src/ansiblelint/rules/only_builtins.py::test_only_builtin_fail
+ --deselect test/test_formatter_json.py::test_code_climate_parsable_ignored
+ --deselect test/test_formatter_sarif.py::test_sarif_parsable_ignored
  --deselect test/test_import_playbook.py::test_task_hook_import_playbook
- --deselect test/test_rules_collection.py::test_rich_rule_listing"
+ --deselect test/test_list_rules.py::test_list_rules_includes_opt_in_rules
+ --deselect test/test_list_rules.py::test_list_rules_with_format_option
+ --deselect test/test_list_rules.py::test_list_tags_includes_opt_in_rules
+ --deselect test/test_main.py::test_call_from_outside_venv
+ --deselect test/test_profiles.py::test_profile_listing
+ --deselect test/test_task_includes.py::test_included_tasks
+ --deselect test/test_utils.py::test_cli_auto_detect
+ --deselect test/test_verbosity.py::test_default_verbosity"
```

Comment:
This adds a few newly failing tests, and mostly reorders the existing ones with `sort -u` so that changes here in the future become easier to reason about